src/components/Rockets.js

```diff
@@ -0,0 +1,59 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { reserveRocket, cancelReservation } from '../redux/rockets/rockets';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.reducerRockets);
+
+  const dispatch = useDispatch();
+
+  const reserveCancelRocket = (id, status) => {
+    if (status === true) {
+      dispatch(cancelReservation(id));
+    } else {
+      dispatch(reserveRocket(id));
+    }
+  };
+
+  const btnReserve = {
+    color: 'white',
+    backgroundColor: '#007bff',
+    border: 'none',
+  };
+
+  const btnCancelation = {
+    color: '#6d757d',
+    backgroundColor: 'white',
+    border: '1px solid #6d757d',
+  };
+
+  const badge = {
+    backgroundColor: '#18a2b8',
+    color: '#fff',
+    padding: '2px 5px',
+    borderRadius: '5px',
+    marginRight: '10px',
+  };
+
+  return (
+    <div className="rockets-list">
+      {rockets.map((rock) => (
+        <div className="rocket-element" key={rock.id}>
+          <div className="image-content">
+            <img src={rock.flickr_images[0]} alt={rock.rocket_name} />
+          </div>
+
+          <div className="rocket-info">
+            <h3>{rock.rocket_name}</h3>
+            <p>
+              <span style={rock.reserved === true ? badge : null}>{rock.reserved === true ? 'Reserved' : null}</span>
+              {rock.description}
+            </p>
+            <button style={rock.reserved === true ? btnCancelation : btnReserve} type="submit" onClick={() => reserveCancelRocket(rock.id, rock.reserved)}>{rock.reserved === true ? 'Cancel Reservation' : 'Reserve Rocket'}</button>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/156244653-5599d59b-03ea-4689-aeb5-e6565b48134c.png)

Therefore:

- [ ] Please, kindly use the React conditional rendering syntax, instead of using ternary operators.

## Hints

You could build if-else statements using this syntax by using the `!` (NOT) operator like this: `{!mission.reserved && (what to do if the previous thing is NOT true)}` and still meet the requirement 💪